DMCompiler/Compiler/DM/DMParserHelper.cs

```diff
@@ -283,13 +283,12 @@ private DMASTExpression ExpressionFromString(Token constantToken)
                                     //Plurals, ordinals, etc
                                     //(things that hug, as a suffix, the [] that they reference)
                                     case "s":
-                                        unimplemented = true;
```

Comment:
These are the last two uses of `unimplemented`. This variable and check below should be removed entirely.